### DISCUSSION TITLE
How to enable experimental WindowRotation when using software renderer as backend

### DISCUSSION THREAD

**Discussion by ierturk** [slint-ui/slint#4030] [Q&A] [answered]
Hi,
There is a flag for the rotating the screen in software renderer. As far as I understand this is an experimental feature. How to enable the feature `software-renderer-rotation` ?

Thank in advance.

https://github.com/slint-ui/slint/blob/9bc8d203fe772dba8834bdaaa7c3d7eefd759414/internal/core/software_renderer.rs#L92
https://github.com/slint-ui/slint/blob/9bc8d203fe772dba8834bdaaa7c3d7eefd759414/internal/core/Cargo.toml#L38

**Comment by ogoffart** [comment, marked as answer]
Add this to your Cargo.toml

```toml
i-slint-core = { version = "=1.3.1", default-features = false, features = ["software-renderer-rotation"] }
```

But please note that by enabling this, you depend on private API that may change from one version to the next.

**Reply by ierturk** [reply]
Thank you
